src/main/java/core/basesyntax/Diamond.java

```diff
@@ -3,16 +3,26 @@
 import java.io.PrintStream;
 
 public class Diamond {
-
-    /**
-     * Нарисовать ромб со звёздочек заданной величины.
-     *
-     * Пример: величина 3, значит максимальное кол-во звёзд в ряду (по диагонали) будет 3.
-     *
-     * Вместо System.out.print() или System.out.println() использовать stream.print()
-     * или stream.println()
-     */
     public static void drawDiamond(int maxSize, PrintStream stream) {
```

Comment:
change this method, and then code will pass test
```suggestion
    protected static void drawDiamond(int maxSize, PrintStream stream) {
```